website/pages/consul-on-kubernetes/index.tsx

```diff
@@ -231,6 +232,38 @@ export default function ConsulOnKubernetesPage() {
       </section>
 
       {/* get started section */}
+      <section>
+        {/* card list */}
+        <DocsList
+          title="Documentation"
+          docs={[
+            {
+              icon: {
+                src: require('./images/docs/helm-icon.png'),
+                alt: 'helm',
+              },
+              description:
+                'Consul offers an official Helm chart for quickly deploying and upgrading Consul on Kubernetes.',
+              cta: {
+                text: 'Heml Docs',
+                url: '#TODO',
+              },
+            },
+            {
+              icon: {
+                src: require('./images/docs/terraform-icon.svg'),
```

Comment:
We have a package for all of our logos that you can use.  This is a bit of a defensive approach to if our logos ever change so we don't have to go scout out every place we copied a version of it (which has happened before).

```suggestion
                src: require('@hashicorp/mktg-logos/product/terraform/logomark/color.svg'),
```